src/SeedCppCore/DAL/Translators/JSON/PatientJSONTranslator.cpp

```diff
@@ -21,7 +21,12 @@ namespace seed_cpp { namespace dal {
 		jsonPatient.addMember("surname", m_patient.getSurname());
 		jsonPatient.addMember("name", m_patient.getName());
 		jsonPatient.addMember("email", m_patient.getEmail());
-		jsonPatient.addMember("dob", boost::posix_time::to_iso_string(m_patient.getDob()));
+		
+		auto dob = m_patient.getDob();
+		if (dob)
+		{
+			jsonPatient.addMember("dob", boost::posix_time::to_iso_string(*m_patient.getDob()));
```

Comment:
I would use the _boost::posix_time::from_iso_extended_string_ method to fix the compatibility issue for the dates of the API. This issue is already fixed in unittests branch.